lab1.js

```diff
@@ -67,6 +69,15 @@ var sentence1 = "More food please.",
  **two** different kinds of loops to implement this.
  HINT: the "split" method on String will be useful.
 */
+//with a little help from our friends in slack
+function chirpMakerTwo(sentence){
+  sentence = sentence.split(" ");
+  for (var i = 0; i < sentence.length; i++) {
+    sentence[i] = "chirp";
+  }
+ return sentence =  sentence.join(" ");
+
+}
```

Comment:
Great. Here's a few things to fix:
1.) For this code, you've declared the function, and now you need to actually run the function and update the value of sentence 1. Remember, you're trying to get the assertions (below) to pass. If you do that, you're close to getting the assertions to pass - you'll just need to get the period on the end of the sentence.
2.) Also, the question is looking for two different kinds of loops here, so you'll need to create another function (say, chirpMakerOne) that uses a different loop (maybe a while loop?) to get the same result.